diff --git a/node_modules/distube/dist/core/DisTubeStream.js b/node_modules/distube/dist/core/DisTubeStream.js
index ddc8db0..9d896e1 100644
--- a/node_modules/distube/dist/core/DisTubeStream.js
+++ b/node_modules/distube/dist/core/DisTubeStream.js
@@ -34,9 +34,9 @@ class DisTubeStream {
         this.url = url;
         /**
          * Stream type
-         * @type {DiscordVoice.StreamType.Raw}
+         * @type {DiscordVoice.StreamType.OggOpus}
          */
-        this.type = voice_1.StreamType.Raw;
+        this.type = voice_1.StreamType.OggOpus;
         const args = [
             "-reconnect",
             "1",
@@ -55,7 +55,7 @@ class DisTubeStream {
             "-ac",
             "2",
             "-f",
-            "s16le",
+            "opus",
         ];
         if (typeof options.seek === "number" && options.seek > 0) {
             args.unshift("-ss", options.seek.toString());
